plugins/presentation/src/components/internal/Table/utils.ts

```diff
@@ -0,0 +1,9 @@
+export function stringFilter (row: object, text: string): boolean {
+  for (const [key, value] of Object.entries(row)) {
+    if (key.startsWith('_')) continue // makes sure it not searching in the private fields
```

Comment:
Better to go through attributes and check if they from Doc, VDoc to not filter. Since there is no term for private or public fields in model right now.

Reply:
I'm not sure if we need this, we are gonna use the search via model, aren't we?